### PR TITLE
Add SectionHeading component

### PR DESCRIPTION
**Pull request by gustaveen**
## Describe your changes
Add SectionHeading component

## Justify why they are needed
Reuse in purchase form
## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by gustaveen** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4844?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#4845** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4845?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4844** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4844?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#4840** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4840?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4838** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4838?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @gustaveen and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: apps/store/src/features/priceCalculator/SectionHeading.tsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    {children}
+  </Heading>
+)
+export const Subtitle = ({ children, ...props }: TitleProps) => (
+  <Heading as="h3" balance={true} color="textSecondary" variant="standard.24" {...props}>
+    {children}
+  </Heading>
+)
+
+export const SectionHeading = {
```
Comment (alebedev) [inline apps/store/src/features/priceCalculator/SectionHeading.tsx:17]: It it worth it for 2 child elements?  Could have been `SectionTitle` and `SectionSubtitle` that are used directly?

**Review by alebedev — APPROVED** [review]

File: apps/store/src/features/priceCalculator/SectionHeading.tsx (hunk 1 above)
Comment (gustaveen) [reply apps/store/src/features/priceCalculator/SectionHeading.tsx:17]: I agree, let's change it 👍 @alebedev

**Comment by gustaveen** [comment]
### Merge activity

* **Oct 10, 7:40 AM EDT**: A user merged this pull request with [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4844).

**Pull request merged**